src/main/java/org/folio/utils/ExceptionUtil.java

```diff
@@ -18,4 +29,42 @@ public static boolean matches(Throwable cause, ErrorCodes errorCode) {
     return false;
   }
 
+  public static Response buildErrorResponse(Throwable throwable) {
+    return buildErrorResponse(ExceptionUtil.handleProcessingError(throwable));
+  }
+
+  private static Response buildErrorResponse(HttpException exception) {
+    final Response.ResponseBuilder responseBuilder;
+    switch (exception.getCode()) {
+      case 400:
+      case 403:
+      case 404:
+      case 409:
+      case 413:
+      case 422:
+        responseBuilder = Response.status(exception.getCode());
+        break;
+      default:
+        responseBuilder = Response.status(INTERNAL_SERVER_ERROR);
```

Comment:
it would be great, if changed this place to use java 17 style in the scope of this pr

Reply:
done